frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -422,7 +422,7 @@ const ManagePolicyPage = ({
           {!!teamId && teamPoliciesError && <TableDataError />}
           {!!teamId &&
             !teamPoliciesError &&
-            (isLoadingTeamPolicies ? (
+            (isLoadingTeamPolicies && isLoadingFailingPoliciesWebhook ? (
```

Comment:
This line should fix detachment issues on e2e test:
     CypressError: Timed out retrying after 20050ms: `cy.click()` failed because this element is detached from the DOM.